src/index.js

```diff
@@ -33,6 +33,13 @@ export default class Firedux {
     if (options.url) {
       console.warn('Firedux option "url" is deprecated, use "ref" instead.')
     }
+
+    this.v3 = options.ref !== null
```

Comment:
Should use `Firebase.SDK_VERSION` checking for `'3.'` prefix. We support `ref` option in both versions.